### PR TITLE
[WIP] OAuth2 Implicit

### PR DESCRIPTION
**Pull request by barroco**

### CODE REVIEW

**Comment by barroco** [comment]
PR #4 before merging this one.

File: routes/oauth2/index.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+var ClientPasswordStrategy = require('./strategy/client-password').client_password_strategy;
+var BearerStrategy = require('./strategy/bearer').bearer;
+var AuthorizationCodeGrant = require('./grant/authorization-code').authorization_code;
+var TokenGrant = require('./grant/token').token;
+var AuthorizationCodeExchange = require('./exchange/authorization-code').authorization_code;
+
+
+module.exports = function (router) {
+
+    var logger = router.get('logger');
```
Comment (chrisn) [inline routes/oauth2/index.js:19]: Use `var logger = require('../../lib/logger');` to get the logger object.

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM ubuntu
+FROM ubuntu:trusty
```
Comment (chrisn) [inline Dockerfile:1]: Does this install the latest stable Node.js version (currently 4.4.4)? Should we use the official Node.js Docker image [here](https://hub.docker.com/_/node/)?

**Comment by barroco** [comment]
No activity for long time, closing.

**Pull request closed without merging**